debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnectorConfig.java

```diff
@@ -343,7 +343,7 @@ public static SecureConnectionMode parse(String value, String defaultValue) {
                                                  .withDescription("Unique name that identifies the database server and all recorded offsets, and"
                                                          + "that is used as a prefix for all schemas and topics. "
                                                          + "Each distinct MySQL installation should have a separate namespace and monitored by "
-                                                         + "at most one Debezium connector. Defaults to 'host:port'");
+                                                         + "at most one Debezium connector.");
```

Comment:
Ah, I changed it because it's defined with `withValidation(Field::isRequired)` so I couldn't omit it in tests. The actual fix then would be to remove this validation I suppose.